docs/intro/tutorial01.txt

```diff
@@ -537,16 +537,11 @@ Now, run :djadmin:`migrate` again to create those model tables in your database:
 .. code-block:: bash
 
     $ python manage.py migrate
-
     Operations to perform:
-      Synchronize unmigrated apps: sessions, admin, messages, auth, staticfiles, contenttypes
-      Apply all migrations: polls
-    Synchronizing apps without migrations:
-      Creating tables...
-      Installing indexes...
+      Apply all migrations: admin, contenttypes, polls, auth, sessions
     Running migrations:
-      Applying polls.0001_initial... OK
-
+      Rendering model states... DONE
+      Applying <migration name>... OK
```

Comment:
Sure, I'll just include the polls migration so we needn't worry if something changes with respect to what's in contrib.